providers/gcp/provider/provider.go

```diff
@@ -166,20 +192,35 @@ func (s *Service) Connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 // It is not necessary to implement this method.
 // If you want to do some cleanup, you can do it here.
 func (s *Service) Shutdown(req *plugin.ShutdownReq) (*plugin.ShutdownRes, error) {
+	for i := range s.runtimes {
+		runtime := s.runtimes[i]
+		// FIXME: I think, we might need the asset here to cleanup the correct connection
```

Comment:
I think, I'll need this for the gcp instance scanning via snapshot. This will be the next PR, as the command still needs migration.